docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -161,6 +161,12 @@ BigQuery       Trino                        Notes
 ``TIMESTAMP``  ``TIMESTAMP_WITH_TIME_ZONE`` Time zone is UTC
 =============  ============================ =============================================================================================================
 
+System Tables
+-------------
+
+The connector supports queries of the view definition. Given a view ``customer_view``,
+``SELECT * customer_view$big_query_views`` shows the view definition.
```

Comment:
How about `customer_view$view_definition`, instead?

Reply:
Looks good. Changed to the new name. Let me remove `syntax-needs-review` now.